cadence/templates/server-configmap.yaml

```diff
@@ -8,83 +8,98 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
 data:
-  docker_template.yaml: |-
+  docker_template_cassandra.yaml: |-
     log:
-        stdout: true
-        level: "${LOG_LEVEL}"
+      stdout: true
+      level: "${LOG_LEVEL}"
 
     persistence:
-        defaultStore: cass-default
-        visibilityStore: cass-visibility
-        numHistoryShards: ${NUM_HISTORY_SHARDS}
-        datastores:
-            cass-default:
-                cassandra:
-                    hosts: "${CASSANDRA_SEEDS}"
-                    keyspace: "${KEYSPACE}"
-                    consistency: "${CASSANDRA_CONSISTENCY}"
-            cass-visibility:
-                cassandra:
-                    hosts: "${CASSANDRA_SEEDS}"
-                    keyspace: "${VISIBILITY_KEYSPACE}"
-                    consistency: "${CASSANDRA_CONSISTENCY}"
+      defaultStore: cass-default
+      visibilityStore: cass-visibility
+      numHistoryShards: ${NUM_HISTORY_SHARDS}
+      datastores:
+        cass-default:
+          cassandra:
+            hosts: "${CASSANDRA_SEEDS}"
+            keyspace: "${KEYSPACE}"
+            consistency: "${CASSANDRA_CONSISTENCY}"
+        cass-visibility:
+          cassandra:
+            hosts: "${CASSANDRA_SEEDS}"
+            keyspace: "${VISIBILITY_KEYSPACE}"
+            consistency: "${CASSANDRA_CONSISTENCY}"
 
     ringpop:
-        name: cadence
-        bootstrapMode: hosts
-        bootstrapHosts: ${RINGPOP_SEEDS_JSON_ARRAY}
-        maxJoinDuration: 30s
+      name: cadence
+      bootstrapMode: hosts
+      bootstrapHosts: ${RINGPOP_SEEDS_JSON_ARRAY}
+      maxJoinDuration: 30s
 
     services:
-        frontend:
-            rpc:
-                port: {{ .Values.server.frontend.service.port }}
-                bindOnIP: {{ .Values.server.frontend.bindOnIP }}
+      frontend:
+        rpc:
+          port: {{ .Values.server.frontend.service.port }}
+          bindOnIP: {{ .Values.server.frontend.bindOnIP }}
+        metrics:
+          prometheus:
```

Comment:
Feels like everybody would love to have Prometheus metrics? Other possible options are `statsd` or `m3` but I would give that up, what you think?